recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - libblas  # [unix]
     - libcblas  # [unix]
     - liblapack  # [unix]
-    - blas 1.1  # [win]
+    - blas  # [win]
```

Comment:
there is no blas package, only libblas

Reply:
Sorry I don’t follow. What does that mean? Btw, for the original my change, I was following https://conda-forge.readthedocs.io/en/latest/blas.html and the example in suitesparse-feedstock. I am pretty much just “monkey see monkey do”.

Reply:
You dont need this since ceres doesnt care about the implementation: libblas is enough.

Reply:
Ah, I see. Just a thought, maybe for easier maintenance, we can remove the `libblas`, `libclas`, and 'liblapack` all together? They are deps from `suitesparse` anyway.